contracts/Staking.sol

```diff
@@ -830,7 +838,7 @@ contract Staking is Pauser, Whitelist {
                 _proposedVotingPowers[i] = validators[bondedValAddrs[i]].tokens;
```

Comment:
We may need to divide the tokens so that the value is in uint64 (maximum of voting power).

Perhaps just div 1e18 (as we assume minimal stake is also 1e18)?  This should be able to support 18e18 tokens (where we are expected to have 100M at the beginning).

Reply:
Looks like the problem is not yet solved?